docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1556,7 +1556,6 @@ Office.NotificationMessageAction:interface:
     });
 
     // Retrieve contextData.
-    // IMPORTANT: At present, this property is in preview and only works in Outlook on Windows.
     Office.context.mailbox.item.getInitializationContextAsync(function (asyncResult) {
         console.log("Initialization context:");
         console.log(asyncResult);
```

Comment:
nit: Can you end the statement on line 1565 with a semicolon so it would be `console.log("a:");`?